inst/www/shared/shiny_scss/shiny.scss

```diff
@@ -469,3 +469,9 @@ html.autoreload-enabled #shiny-disconnected-overlay.reloading {
   /* override anything bootstrap sets for `.nav` */
   display: none !important;
 }
+
+/* Conditional panels and uiOutput */
+[data-display-if],
```

Comment:
I'd rather you add an explicit class for conditionalPanels. I like the idea that you could use `data-display-if` directly on any element if you want to, and have it show/hide.

Reply:
Done in e93dd1e797574f5c729819453f9301febdf4e5f3